app/models/user/notifications/donation_subscription_created_notification.rb

```diff
@@ -0,0 +1,19 @@
+class User
+  module Notifications
+    class DonationSubscriptionCreatedNotification < Notification
+      params :payment
+
+      def url
+        Exercism::Routes.settings_url
+      end
+
+      def image_type; end
+
+      def image_url; end
+
+      def guard_params
+        "Badge##{user_acquired_badge.badge_id}"
```

Comment:
Why does the guard here use the badge ID? Is that for uniqueness?

Reply:
This file should have been deleted :+1: